scripts/start.sh

```diff
@@ -73,7 +73,6 @@ function main() {
   case $ARG in
     mysql)
       setup_mysql
-      sleep infinity
```

Comment:
workflow test using this script to setup a MySQL service on Kubernetes
https://github.com/sql-machine-learning/sqlflow/blob/4a09c55218051da3940fb0759f02d1e8e3417bc0/scripts/test/workflow.sh#L77

Please also update the command spec to let MySQL Pod as a long time service: 
``` bash
    kubectl run mysql --port 3306 --env="SQLFLOW_MYSQL_HOST=0.0.0.0" --env="SQLFLOW_MYSQL_PORT=3306" --image=${SQLFLOW_WORKFLOW_STEP_IMAGE} --command -- bash -c 'bash /start.sh mysql; sleep sleep infinity'
```

Reply:
thanks @Yancey1989 a lot for the review! I found that I should not use ./start.sh mysql, instead, the populate-example-dataset-mysql should be used. I'm closing this PR.